### ISSUE TITLE
aocr test does not work for real picture?

### ISSUE THREAD

**Issue by nickppbd** [emedvedev/attention-ocr#67]
80% of http://www.cs.cmu.edu/~yuntiand/sample.tgz for train

the rest of it for testing

I trained and made model(model.ckpt-9000.data-00000-of-00001, model.ckpt-9000.index, model.ckpt-9000.meta) and aocr test command for the rest of the link sample data.

The result was great!, so I decided to aocr test command for my own picture.

I took a picture of a word(it was 'months' word) and edit size(to 187 x 31) and set background color to the same of the picture at ADOBE PHOTOSHOP![months](https://user-images.githubusercontent.com/2631139/35893891-7f254b18-0bf3-11e8-8003-ca9636e46262.jpg).(The reason I set the background color is aocr test did not work at the image which was applied only for size edit, so I just tried)

Anyway it did not work at all. I aocr dataset and make testing.tfrecords file and tried below and it does not make results folder and there is no step process.

C:\Users\kimduknam>aocr dataset ./datasets/annotations-testing.txt ./datasets/testing.tfrecords
2018-02-07 10:37:33.179317: I C:\tf_jenkins\workspace\rel-win\M\windows\PY\36\tensorflow\core\platform\cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2018-02-07 10:37:33,182 root  INFO     Building a dataset from ./datasets/annotations-testing.txt.
2018-02-07 10:37:33,182 root  INFO     Output file: ./datasets/testing.tfrecords
2018-02-07 10:37:33,182 root  INFO     Processed 1 pairs.
2018-02-07 10:37:33,182 root  INFO     Dataset is ready: 1 pairs.
2018-02-07 10:37:33,182 root  INFO     Longest label (6): MONTHS

C:\Users\kimduknam>aocr test --max-prediction 30 --visualize ./datasets/testing.tfrecords
2018-02-07 10:37:39.292842: I C:\tf_jenkins\workspace\rel-win\M\windows\PY\36\tensorflow\core\platform\cpu_feature_guard.cc:137] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2018-02-07 10:37:39,294 root  INFO     phase: test
2018-02-07 10:37:39,294 root  INFO     model_dir: ./checkpoints
2018-02-07 10:37:39,294 root  INFO     load_model: True
2018-02-07 10:37:39,294 root  INFO     output_dir: ./results
2018-02-07 10:37:39,294 root  INFO     steps_per_checkpoint: 0
2018-02-07 10:37:39,294 root  INFO     batch_size: 1
2018-02-07 10:37:39,294 root  INFO     learning_rate: 1
2018-02-07 10:37:39,294 root  INFO     reg_val: 0
2018-02-07 10:37:39,294 root  INFO     max_gradient_norm: 5.000000
2018-02-07 10:37:39,294 root  INFO     clip_gradients: True
2018-02-07 10:37:39,294 root  INFO     max_image_width 160.000000
2018-02-07 10:37:39,294 root  INFO     max_prediction_length 30.000000
2018-02-07 10:37:39,294 root  INFO     channels: 1
2018-02-07 10:37:39,294 root  INFO     target_embedding_size: 10.000000
2018-02-07 10:37:39,294 root  INFO     attn_num_hidden: 128
2018-02-07 10:37:39,294 root  INFO     attn_num_layers: 2
2018-02-07 10:37:39,294 root  INFO     visualize: True
2018-02-07 10:37:42,850 root  INFO     Reading model parameters from ./checkpoints\model.ckpt-9000

**Comment by nickppbd** [comment]
I'm sorry it's just max_image_width problem. I resize months.jpg to 159x29. It works well!
